crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -1222,7 +1222,7 @@ impl<F: Clone> UpdateTracker<F, PaymentData<F>, api::PaymentsRequest> for Paymen
         let order_details = payment_data.payment_intent.order_details.clone();
         let metadata = payment_data.payment_intent.metadata.clone();
         let frm_metadata = payment_data.payment_intent.frm_metadata.clone();
-        let authorized_amount = payment_data
+        let mut authorized_amount = payment_data
```

Comment:
do not use mutable variables, whatever calculation is necessary to be done, try to do it when initializing the variable